common/ideas/02_zael_national_ideas.txt

```diff
@@ -0,0 +1,213 @@
+tunis_ideas = {
+	start = {
+		galley_power = 0.25
+		may_perform_slave_raid = yes
+	}
+	
+	bonus = {
+		naval_forcelimit_modifier = 0.3
+	}
+	
+	trigger = {
+		tag = TUN
+	}
+	
+	catalan_guard = {
+		infantry_power = 0.15
+	}
+	dual_diplomacy = {
+		rival_change_cost = -1
+		diplomatic_upkeep = 1
+	}
+	corsairs = {
+		navy_tradition = 1
+		privateer_efficiency = 0.25
+	}
+	attract_foreign_pirates = {
+		global_sailors_modifier = 0.25
+		regiment_manpower_usage = -0.15
+	}
+	board_of_captains = {
+		leader_navy_fire = 1
+		leader_army_fire = 1
+	}
+	export_monopolies = {
+		trade_efficiency = 0.15
+		embargo_efficiency = 0.25
+	}
+	tunisian_caliphate = {
+		own_tolerance = 2
+		monthly_piety_accelerator = 0.001
+	}
+}
+
+scotland_ideas = {
+	start = {
+		land_forcelimit_modifier = 0.33
+		leader_land_shock = 1
+	}
+	
+	bonus = {
+		land_morale = 0.15
+	}
+	
+	trigger = {
+		tag = SCO
+	}
+	
+	break_the_black_douglas = {
+		global_unrest = -2
+		tax_efficiency = 0.15
+	}
+	modernize_the_royal_scots_navy = {
+		global_ship_cost = -0.1
+	}
+	rein_in_the_clans = {
+		global_manpower_modifier = 0.2
+	}
+	comets = {
+		stability_cost_modifier = -0.1
+	}
+	basilikon_doron = {
+		legitimacy = 1
+		devotion = 1
+		republican_tradition = 0.3
+	}
+	pass_the_act_of_revocation = {
+		ae_impact = -0.15
+	}
+	remember_the_bruce = { # get stuck in ya blaggards
+		infantry_shock = 0.5
+	}
+}
```

Comment:
I assume we're still attempting to do something like a "Celtic Britain"? Because if not, the ideas seem mid for a country intending to play tall, non-colonial Britain

Reply:
Says the man who banned Caroleans ;p

yeah i could put stability down with basilikon doron and put another mil modifier there. or maybe put the "remember_the_bruce" there, and put something else at the end? either way i'm cool with it.

Naval buffs will come from mission "Rule Britannia" so no need to worry about that. We could also just increase infantry shock from 0.5 to 1.0 or something?

Reply:
I'm never approving a merge that has Scottish caroleans

I think we want Scottish Britain that's highly devved and defensible, not impossible to defeat.

I was thinking more regarding trade/economic/society/tech buffs

Reply:
They get iirc -5% idea cost -5% tech cost from a mission.

I'll add max absolutism, reform progress, and something trade related